proto/panacea/datadeal/v2/deal.proto

```diff
@@ -17,6 +17,7 @@ message Deal {
   string consumer_address = 7;
   repeated AgreementTerm agreement_terms = 8;
   DealStatus status = 9;
+  bytes presentation_definition = 10;
```

Comment:
Let's move this one to right below the `data_schema = 3` for readability. For now, we don't need to care that much of backward compatibility since we're not on mainnet yet.

Reply:
I see. :+1: